### ISSUE TITLE
[Feat Req] Email forward with no mailbox

### ISSUE THREAD

**Issue by Chewza** [sentora/sentora-core#114]
Is it possible to modify the email forwarder process to not include a mailbox to be created first?
It seems counter-intuitive to be required to create a mailbox, then go back and forward the mail from that mailbox to another account.

It's nothing that's critical to the functionality of the system, however for end users coming from other control panels, it's a little cumbersome to try and figure out.

**Comment by bobsta63** [comment]
Isn't this just the 'Alias' feature?

**Comment by Chewza** [comment]
No, the alias feature requires a mailbox to be available locally.
I'm trying to forward mail from a domain on Sentora to Gmail.
I've gotten around the issue by creating distribution lists with 1 user in them which don't require a mailbox to be configured, however it seems counterintuitive to set it up that way.

**Comment by MBlagui** [comment]
True... so mainly to setup forwarding to external emails instead of internal emails. Postfix support it.
